### ISSUE TITLE
 html-result is not output

### ISSUE THREAD

**Issue by cha2maru** [vmi/selenese-runner-java#277]
As a result of executing Selenese-Runner in the following environment, html-result was not output.

- Windows7 64bit

```
java -jar selenese-runner.jar --driver chrome --chromedriver ./external/webdriver/bin/chromedriver.exe --html-result ./html-result --xml-result ./xml-result --screenshot-dir ./screenshot ./TestScript/TestSuite.html
```

As a result of executing in Seleneserunner 3.14 in the same environment, html-result is outputted, so it seems to be due to some degradation or. Side correspondence.

I would be pleased if you confirm it when you have time.
Thanks.

**Comment by vmi** [comment]
Hi,

Thank you for the report.
I released 3.17.0 which fixed this problem, so please check it.

**Comment by cha2maru** [comment]
Thank you for the quick answer.
I confirmed that this problem is solved with the latest version.
Close this Issue.
